backend/danswer/search/pipeline.py

```diff
@@ -169,36 +171,39 @@ def _get_sections(self) -> list[InferenceSection]:
         above = self.search_query.chunks_above
         below = self.search_query.chunks_below
 
-        functions_with_args: list[tuple[Callable, tuple]] = []
+        # functions_with_args: list[tuple[Callable, tuple]] = []
         expanded_inference_sections = []
 
         # Full doc setting takes priority
+
         if self.search_query.full_doc:
             seen_document_ids = set()
             unique_chunks = []
+            list_inference_chunks = []
             # This preserves the ordering since the chunks are retrieved in score order
             for chunk in retrieved_chunks:
                 if chunk.document_id not in seen_document_ids:
                     seen_document_ids.add(chunk.document_id)
                     unique_chunks.append(chunk)
-
-                    functions_with_args.append(
-                        (
-                            self.document_index.id_based_retrieval,
-                            (
-                                chunk.document_id,
-                                None,  # Start chunk ind
-                                None,  # End chunk ind
-                                # There is no chunk level permissioning, this expansion around chunks
-                                # can be assumed to be safe
-                                IndexFilters(access_control_list=None),
-                            ),
-                        )
-                    )
-
-            list_inference_chunks = run_functions_tuples_in_parallel(
-                functions_with_args, allow_failures=False
-            )
+                    list_inference_chunks.append([chunk])
+
+                    # NOTE: left in for eventual refactor with Vespa API
```

Comment:
can we leave this all the same as before? With this, it won't actually use full docs so the behavior will just be incorrect, right?